python/ray/tests/test_metrics.py

```diff
@@ -300,7 +300,8 @@ class ParentActor:
         def __init__(self):
             self.a = [ActorRequiringGPU.remote() for i in range(4)]
 
-    ParentActor.remote()
+    parent_actor = ParentActor.remote()
+    assert parent_actor is not None
```

Comment:
I don't think you need this line.

Reply:
This line is to solve the lint error: local variable 'parent' actor 'is assigned to but never used